### PR TITLE
Task2

### PR DESCRIPTION
**Pull request by VolhaZha**
Task 2

### CODE REVIEW

File: Order.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    internal class Order
+    {
+        private string name;
+        private long phone;
+        private float price;
+        private string address;
+
+        public string Name { get { return name; } set { name = value; } }
+        public long Phone { get { return phone; } set { phone = value;
+                string phoneString = value.ToString();
```
Comment (IgorKheidorov) [inline Order.cs:20]: Magic numbers

File: Order.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
+﻿namespace OzTest1
+{
+    internal class Order
+    {
```
Comment (IgorKheidorov) [inline Order.cs:4]: private const long PNONE_LENGTH = 13;

File: Order.cs [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        private string name;
+        private long phone;
+        private float price;
+        private string address;
+
+        public string Name { get { return name; } set { name = value; } }
+        public long Phone { get { return phone; } set { phone = value;
+                string phoneString = value.ToString();             
+                if (phoneString.Length != 13)
```
Comment (IgorKheidorov) [inline Order.cs]: if (phoneString.Length != PHONE_LENGTH)

**Review by IgorKheidorov — CHANGES_REQUESTED** [review]
Exceptions

**Comment by VolhaZha** [comment]
No need.

**Pull request closed without merging**